packages/elements/src/components/ino-autocomplete/ino-autocomplete.scss

```diff
@@ -1,41 +1,69 @@
 @use '../base/mdc-customize';
 @use '@material/list';
-@use '../base/theme' as theme;
-@use '../base/new-theme' as new-theme;
+@use '../base/new-theme' as theme;
+@use '../base/option-base' as base;
 
 @include list.deprecated-core-styles;
 
+$option-selected-background-color: #{theme.$p-1};
+$option-selected-background-color-hover: #{theme.$p-2};
+$option-selected-background-color-focus: #{theme.$p-3};
+$option-selected-background-color-active: #{theme.$p-3};
+
+$option-deselected-background-color: #{theme.$white};
+$option-deselected-background-color-hover: #{theme.$n-1};
+$option-deselected-background-color-focus: #{theme.$n-2};
+$option-deselected-background-color-active: #{theme.$n-3};
```

Comment:
Aren't these colors the same for the `ino-option`? If that's the case, we should directly define them in our `_option-base.scss` so we don't have to define the same colors values in our autocomplete and select component

Reply:
I just saw that the colors of the `ino-option` are configurable. In that case, I would define constants in the `_option-base.scss` and use them as the fallback values for our CSS Variables.

Like this:
```scss
// in option-base.scss
$SELECTED_BG_COLOR: #{theme.$p-1}

// in ino-option.scss
@use '../base/option-base' as base;

  --option-selected-background-color: var(
    --ino-option-selected-background-color,
    base.$SELECTED_BG_COLOR
  );
```

That way, we've got all our color states (selected, unselected, active, ...) in the `_option-base.scss` and don't have to repeat them in our `ino-select` or `ino-autocomplete`